query-engine/driver-adapters/src/proxy.rs

```diff
@@ -531,12 +531,12 @@ impl TryFrom<JSResultSet> for QuaintResultSet {
 
 impl CommonProxy {
     pub fn new(object: &JsObject) -> napi::Result<Self> {
-        let flavour: JsString = object.get_named_property("flavour")?;
+        let provider: JsString = object.get_named_property("provider")?;
```

Comment:
```suggestion
        let provider: JsString = object.get_named_property("provider").or_else(|| object.get_named_property("flavour"))?;
```

I think that's about it? (untested).
Feel free to ignore if you disagree.

Reply:
Good idea!